packages/core/src/sass/app/_common/drawer/contract-drawer.scss

```diff
@@ -169,9 +169,9 @@ $header-height: 4em;
         }
     }
     &__symbol {
-        margin-left: 0.5rem;
-        font-size: 1em;
-        line-height: 1.5;
+        margin-left: 0.8rem;
+        font-size: 1.2rem;
+        line-height: normal;
```

Comment:
Do we use this `normal` value anywhere else? IMO we should keep using numbers.

Reply:
Let's resolve this in another card.

Reply:
Sure